src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/impl/IndividualRepoImpl.java

```diff
@@ -264,18 +263,17 @@ private SimpleQuery buildSimpleQuery(List<FilterArg> filters) {
 
     private List<SimpleFilterQuery> buildFilterQueries(List<FilterArg> filters) {
         List<SimpleFilterQuery> results = new ArrayList<SimpleFilterQuery>();
-        Map<String, List<FilterArg>> filtersGrouped = filters.stream().collect(Collectors.groupingBy(w -> w.getField()));
-        filtersGrouped.forEach((field, filterList) -> {
+        filters.stream().collect(Collectors.groupingBy(w -> w.getField())).forEach((field, filterList) -> {
             FilterArg firstOne = filterList.get(0);
-            Criteria crit = new CriteriaBuilder(firstOne).buildCriteria();
-            // the rest (of that field) are Or'd
+            Criteria criteria = new CriteriaBuilder(firstOne).buildCriteria();
+            // the rest (of that field) are AND'd
+            // possible solution is to add another filter value delimiter to allow to specify AND/OR
```

Comment:
What about something more like?
```suggestion
            // this approach is to add another filter value delimiter to allow to specify AND/OR
```

The reason behind my thoughts are in regards to how I read "possible solution" as something that is usually preceded by `@todo` or `@fixme`. I think this is describing the solution rather than a possible solution.

Reply:
Yeah, I `TODO` is also a token that can be used. I was just thinking to have the comment for future reference.